### DISCUSSION TITLE
Umzug / Parallelbetrieb alte Integration/custom_homematic

### DISCUSSION THREAD

**Discussion by ProfDrYoMan** [SukramJ/aiohomematic#413] [Q&A]
Hallo,

hier ist schonmal HACS installiert und alles vorbereitet für die Installation von custom_homematic.

Bei mir hängt allerdings die Steuerung der Heiztherme mit am HA (mit Aktorik am HMIP), deswegen warte ich noch bis die Heizung sicher aus sein kann. Wird ja wohl sehr bald sein.

Nun die Frage: Wie tun. Ich hatte vor Ewigkeiten mal gefragt, Parallelbetrieb sollte tun, wenn irgenwie ein "Basis"-Name eindeutig anders ist, aber ich denke das macht noch anderen Ärger.

Die neuen Devices/Entities werden ja dann mit anderer ID (_2 am Ende) angelegt. Ich weiß nicht ob man das will, denn dann später alles umbenennen ist auch schmerzhaft.

Aber trotzdem, wäre das irgendwie ein gangbarer Weg? Wenn ja, wie?

Sonst, ganz hart: Funktioniert sowas?

- Alte Integration deinstallieren.
- Alle alten Entities löschen (geht das sicher, mir ist oft aufgefallen das ich entities nicht löschen kann).
- custom_homematic installieren.
- Alle Automationen/Szenen/... und "magischen"-Sensoren (für die Heizungsteuerung) anpassen.

Was vergessen?

**Comment by emufan** [comment]
So würde ich es an Deiner Stelle machen.

**Comment by ProfDrYoMan** [comment]
Getan und erfolgt. Läuft seit ca. einer Woche problemlos war aber doch ne ganze Ecke Arbeit.

**Reply by patman15** [reply]
> Getan und erfolgt. Läuft seit ca. einer Woche problemlos war aber doch ne ganze Ecke Arbeit.

Hast du noch Hinweise, auf was man aufpassen sollte, nachdem du ja erfolgreich warst? Hab das aktuell auch vor und würde gerne insbesondere Langzeithistorien nicht verlieren, oder endet man sowieso in Datenbankmanipulationen?

**Reply by ProfDrYoMan** [reply]
Naja, die ids ändern sich halt hier und da. Sonst alles gut. Langzeit war mir alles egal. Ergo ein Bruch.
